.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
+language: python
+python:
+  - "3.5"
+  - "3.6"
+matrix:
+  include:
+    - os: windows
+      language: sh
+      python: "3.7"
+      before_install:
+        - choco install python3
+        - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
+        - choco install saltminion
+        - export PATH="/c/salt:$PATH"
+        - python -m pip install --upgrade pip wheel
+
+install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash ./build/install_linux.sh; fi
```

Comment:
When running scripts individually like this, they need to be written in a way where failures in the script are propagated as non-zero exit codes. Otherwise, we just get false successes from CI.